0-bootstrap/sa.tf

```diff
@@ -35,14 +35,20 @@ locals {
       "roles/resourcemanager.organizationViewer",
       "roles/accesscontextmanager.policyAdmin",
       "roles/essentialcontacts.admin",
+      "roles/browser",
+    ],
+    "env" = [
+      "roles/browser",
```

Comment:
Let's move this to a common_roles local and then concat local here. Then for the common_roles we can add the justification of why we added this and the user can remove if needed.

Reply:
`common_roles` local created with an explanation for the need of `browser` role